cmake/thirdparty/SetupThirdParty.cmake

```diff
@@ -70,6 +70,14 @@ if (ENABLE_CUDA)
   include(${BLT_ROOT_DIR}/cmake/thirdparty/SetupCUDA.cmake)
 endif()
 
+################################
+# ROCM
+################################
+message(STATUS "ROCm Support is ${ENABLE_ROCM}")
```

Comment:
For consistency, it would be best if we used either ``ROCm`` or ``ROCM`` in all output messages.  

(The latter is more consistent with blt.  We've tried to only use mixed case with ``Fortran``, which was imposed by cmake).